### PR TITLE
Add JDK 22 clojure variants

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

File: library/clojure [hunk 1]
```diff
@@ -231,3 +231,65 @@ Directory: target/debian-bullseye-slim-21/tools-deps
...
+Directory: target/debian-bookworm-slim-22/boot
+
+Tags: temurin-22-boot-2.8.3-bullseye, temurin-22-boot-bullseye
+Directory: target/debian-bullseye-22/boot
+
+Tags: temurin-22-boot-2.8.3-bullseye-slim, temurin-22-boot-bullseye-slim
+Directory: target/debian-bullseye-slim-22/boot
+
+Tags: temurin-22-boot-2.8.3-focal, temurin-22-boot-focal
+Directory: target/eclipse-temurin-22-jdk-focal/boot
```
Comment (LaurentGoderre) [inline library/clojure:248]: There is no focal base image for JDK 22

File: library/clojure [hunk 2]
```diff
@@ -231,3 +231,65 @@ Directory: target/debian-bullseye-slim-21/tools-deps
...
+Directory: target/debian-bookworm-slim-22/lein
+
+Tags: temurin-22-lein-2.11.2-bullseye, temurin-22-lein-bullseye
+Directory: target/debian-bullseye-22/lein
+
+Tags: temurin-22-lein-2.11.2-bullseye-slim, temurin-22-lein-bullseye-slim
+Directory: target/debian-bullseye-slim-22/lein
+
+Tags: temurin-22-lein-2.11.2-focal, temurin-22-lein-focal
+Directory: target/eclipse-temurin-22-jdk-focal/lein
```
Comment (LaurentGoderre) [inline library/clojure:270]: Same as previous comment

File: library/clojure [hunk 3]
```diff
@@ -231,3 +231,65 @@ Directory: target/debian-bullseye-slim-21/tools-deps
...
+Directory: target/debian-bookworm-slim-22/tools-deps
+
+Tags: temurin-22-bullseye, temurin-22-tools-deps-1.11.1.1435-bullseye, temurin-22-tools-deps-bullseye
+Directory: target/debian-bullseye-22/tools-deps
+
+Tags: temurin-22-bullseye-slim, temurin-22-tools-deps-1.11.1.1435-bullseye-slim, temurin-22-tools-deps-bullseye-slim
+Directory: target/debian-bullseye-slim-22/tools-deps
+
+Tags: temurin-22-focal, temurin-22-tools-deps-1.11.1.1435-focal, temurin-22-tools-deps-focal
+Directory: target/eclipse-temurin-22-jdk-focal/tools-deps
```
Comment (LaurentGoderre) [inline library/clojure:292]: Same as previous comment

**Comment by cap10morgan** [comment]
@LaurentGoderre Sorry, should have checked that first. Kind of surprising, though.

**Comment by LaurentGoderre** [comment]
@cap10morgan no need to close the PR, you can amend it.

**Comment by cap10morgan** [comment]
@LaurentGoderre Sure, but I don't actually want to release these until they're available from all upstreams. This was from a third-party PR on our end that I didn't check as thoroughly as I should have.

**Pull request closed without merging**